recipe/meta.yaml

```diff
@@ -46,7 +46,7 @@ requirements:
     - scipy >=0.19
     - matplotlib-base >=2.0.0
     - networkx >=2.0
-    - pillow >=4.3.0
+    - pillow >=4.3.0,!=7.1.*
```

Comment:
Should it be?:
```suggestion
    - pillow >=4.3.0,!=7.1.0,!=7.1.1
```

Reply:
Assuming that the next point release has the fix, yes that's better